app/resources/documentation/manual/index.md

```diff
@@ -440,11 +440,13 @@ TrenchBroom gives you various options on how to create new objects. In the follo
 
 ### Creating Simple Shapes
 
-The easiest way to create a new brush is to just draw it with the mouse. To draw a brush, make sure nothing is currently selected and left drag in the 3D viewport or any of the 2D viewports. If you draw a brush in the 3D viewport, its shape is controlled by the point where you initially started your drag, the point currently under the mouse, and the current grid size. Essentially, you will draw your brush on the XY plane, and its height will be set to the current grid size. But it is possible to change the height while drawing a brush by holding #key(Alt). Whenever #key(Alt) is held, you can change the brush height, otherwise you control the X/Y extents of the brush.
+The easiest way to create a new brush is to just draw it out with the mouse using the Simple Shape Tool which is enabled by default when nothing else is selected or any other tool active. Left drag in the 3D viewport or any of the 2D viewports.
+
+When drawing a brush in the 3D viewport, its shape is controlled from the point under the mouse where you initially started your drag, to the point currently under the mouse cursor, and the current grid size. Essentially, while drawing your brush on the XY plane, and its height will be set to the current grid size. While dragging you can force the XY planes to equal lengths by holding #key(Shift), or force XYZ planes to equal lengths by holding #key(Shift)+#key(Alt), and its also possible to change just the height while drawing a brush by holding #key(Alt).
```

Comment:
This sentence isn't right:

> Essentially, while drawing your brush on the XY plane, and its height will be set to the current grid size.

How about:

> When drawing your brush on the XY plane, the height of the brush will be set to the current grid size.

Reply:
This is a bit imprecise:

> While dragging you can force the XY planes to equal lengths by holding #key(Shift), or force XYZ planes to equal lengths by holding #key(Shift)+#key(Alt), and its also possible to change just the height while drawing a brush by holding #key(Alt).

You are not forcing planes to be equal length. What is the length of a plane? I think you mean "axes" here?

> While dragging you can force the X and Y axes to equal lengths by holding #key(Shift), or force all three axes (X, Y and Z) to equal lengths by holding #key(Shift)+#key(Alt). Finally, it's also possible to change just the height while drawing a brush by holding #key(Alt).

Reply:
Yes, that's what was already written previously, but planes should have been axes. I will make that change. The modifiers do force all axes to have equal lengths as you drag out the shape.

![image](https://github.com/user-attachments/assets/eddb05df-1569-4a83-adc0-51f2ab9ef847)

 I will switch planes to axes and drop the *`length`* as it is an extra word unneeded, and leave it just equal or another word for equal in geometry terms would be `congruent`.